package.json

```
@@ -24,7 +24,7 @@
     "snazzy": "^9.0.0",
     "standard": "^17.0.0",
     "tap": "^16.2.0",
-    "unicode-emoji-json": "^0.3.1",
+    "unicode-emoji-json": "^0.4.0",
```

We might as well go ahead and bump to the latest version (since we don't have a way to simply re-run the build and rely on the semver qualifier to do it for us):
```suggestion
    "unicode-emoji-json": "^0.5.0",
```